asciidoc/courses/app-java/modules/1-getting-started/lessons/1-setup/lesson.adoc

```diff
@@ -1,72 +1,62 @@
 
-[TIP]
-.Installing SDKMAN
-====
-If you haven't already done so, you can install https://sdkman.io/[sdkman^] at https://sdkman.io/[sdkman.io^]
-====
+A blank Neo4j Sandbox instance has also been created for you to use during this course.
 
+You can open a Neo4j Browser window throughout this course by clicking the link:#[Toggle Sandbox,role=classroom-sandbox-toggle] button in the bottom right-hand corner of the screen.
 
-The project has been written to work with Java version **{java-version}**.
+== Get the code
 
-You can verify that the installation is successful by running the following command in the command line:
+You can use link:https://gitpod.io[Gitpod^] as an online IDE and workspace for this workshop.
+It will automatically clone the workshop repository and set up your environment.
 
-.Verify Java Version
-[source,sh]
-----
-java --version
-----
+lab::Open `Gitpod workspace`[]
 
-You will also need a package manager to install the dependencies and build and run the project.
-We use Apache Maven with this project.
-Maven is not included with Java, but you can also install it with sdkman.
+[NOTE]
+You will need to login with a Github, Gitlab, or Bitbucket account.
 
-.Install Maven
-[source,sh]
-----
-sdk install maven
-----
+Alternatively, you can clone the repository and set up the environment yourself.
 
+[%collapsible]
+.Develop on your local machine
+====
+1. Verify your Java version. You will need Java 17 or higher. You can check your version by running the following command in your terminal window: +
+`java -version`. To download Java, you can choose from many link:https://neo4j.com/docs/operations-manual/current/installation/requirements/#deployment-requirements-software[supported vendor options^]. For example, link:https://www.azul.com/downloads/?package=jdk#zulu[Azul's JDK^] or hlink:ttps://openjdk.org/install/[OpenJDK^].
 
-// == Clone the Repository
+2. Pick an IDE of your choice. If you don't yet have a preference, you can download link:https://www.jetbrains.com/idea/download[IntelliJ IDEA Community Edition^] or link:https://code.visualstudio.com/download[Visual Studio Code^], are both excellent choices.
+====
 
-include::{shared}/lesson/clone-repository.adoc[]
+You do not need to create a Neo4j database as you will use the provided sandbox instance.
 
+read::Continue[]
 
-== Install Dependencies
+[.summary]
+== Lesson Summary
 
-Once you have cloned the repository, navigate to the folder in your terminal and run the following command to install the dependencies and test the project: `mvn verify`
+In this lesson, you reviewed any prerequisites needed to complete this course and set up your development environment to work with Java and Neo4j.
 
-.Install Dependencies using Maven
-[source,sh]
-----
-mvn verify
-----
-
-You can view a full list of dependencies by opening link:{repository-blob}/main/pom.xml[`pom.xml`^].
+//TODO: Update this!!
+In the next lesson, you will learn about ???.
```

Comment:
I can come back and update this soon?

Reply:
Sure, for the time being I will comment out the line.